eng/common/pipelines/templates/steps/prepare-pipelines.yml

```diff
@@ -35,6 +35,7 @@ steps:
         --agentpool Hosted
         --branch refs/heads/$(DefaultBranch)
```

Comment:
You are going to also need to update https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/pipeline-generation.yml

Reply:
You will also need to update https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/pipelines/templates/steps/install-pipeline-generation.yml so you might need to do these yml files after this PR is merged.